utilities/pipelines/platform/Set-AvmGitHubIssueForWorkflow.ps1

```diff
@@ -0,0 +1,247 @@
+                        $ProjectNumber = 566 # AVM - Module Issues
+                        $comment = @"
+> [!IMPORTANT]
+> @Azure/$($module.ModuleOwnersGHTeam), the workflow for the ``$moduleName`` module has failed. Please investigate the failed workflow run. If you are not able to do so, please inform the AVM core team to take over.
+"@
+                        # assign owner
+                        if ($PSCmdlet.ShouldProcess(('Owner [{0}] to issue [{1}]' -f $module.PrimaryModuleOwnerGHHandle, $issueName), 'Assign')) {
+                            $assign = gh issue edit $issueUrl --add-assignee $module.PrimaryModuleOwnerGHHandle --repo "$RepositoryOwner/$RepositoryName"
+                        }
+                        if ([String]::IsNullOrEmpty($assign)) {
+                            $comment = @"
+> [!WARNING]
+> This issue couldn't be assigend due to an internal error. @$($module.PrimaryModuleOwnerGHHandle), please make sure this issue is assigned to you and please provide an initial response as soon as possible, in accordance with the [AVM Support statement](https://aka.ms/AVM/Support).
+"@
+                            if ($PSCmdlet.ShouldProcess("Missing user comment to issue [$issueName]", 'Add')) {
+                                $userCommentUrl = gh issue comment $issueUrl --body $comment --repo "$RepositoryOwner/$RepositoryName"
+                            }
+                            Write-Verbose ('💬 Commented issue {0} ({1}) as the automation was unable to auto-assign the module owner. ({2})' -f $issueUrl, $issueName, $userCommentUrl) -Verbose
+                        } else {
+                            Write-Verbose ('👋 Assigned owner [@{0}] to issue {1} ({2})' -f $module.PrimaryModuleOwnerGHHandle, $issueUrl, $issueName) -Verbose
+                        }
+                    }
+                }
+
+                if ($PSCmdlet.ShouldProcess("Issue [$issueName] to project [AVM - Issue Triage]", 'Add')) {
+                    $null = Add-GithubIssueToProject -Repo "$RepositoryOwner/$RepositoryName" -ProjectNumber $ProjectNumber -IssueUrl $issueUrl
+                }
+
+                if ($PSCmdlet.ShouldProcess("Comment to issue [$issueName]", 'Add')) {
+                    $commentUrl = gh issue comment $issueUrl --body $comment --repo "$RepositoryOwner/$RepositoryName"
+                }
+                Write-Verbose ('💬 Commented issue {0} ({1}) as its module''s latest run in the main branch failed. ({2})' -f $issueUrl, $issueName, $commentUrl) -Verbose
+
+                $issuesCreated++
+            } else {
+                # If an issue does already exist, add a comment to it
+                $issueToComment = @() + ($issues | Where-Object {
+                        $_.title -eq $issueName
+                    })
+
+                # Handling duplicated issues (which should be cleaned up)
```

Comment:
Not for this PR but as a next improvement what about closing as not planned and adding a duplicate label?

Reply:
done